src/briefcase/bootstraps/pyside6.py

```diff
@@ -60,6 +60,7 @@ def pyproject_table_briefcase_app_extra_content(self):
     def pyproject_table_macOS(self):
         return """\
 universal_build = true
+min_os_version = "12.0"
 requires = [
     "std-nslog~=1.0.3",
 ]
```

Comment:
Think it's worth a comment explaining why macOS 11 isn't supported?

Reply:
Good call; added.